azure/src/services/azureFunctionCloudService.test.ts

```diff
@@ -131,7 +163,14 @@ describe("Azure Cloud Service should", () => {
       ...axiosRequestConfig,
       headers: headers.toJSON()
     };
-    const response = cloudService.invoke("azure-getCart", false, null, headers);
+
+    const params: InvokeRequest = {
+      name: "azure-getCart",
+      fireAndForget: false,
+      payload: null,
+      headers: headers
+    };
+    const response = cloudService.invoke(params);
```

Comment:
Shouldn't this statement be awaitable?

Reply:
Yes it should, thanks for this feedback, it will be added in my next commit